.yamato/config/postprocessing.metafile

```diff
@@ -4,8 +4,8 @@ project:
 test_platforms:
   - type: playmode
     allow_cache: False
-  - type: editmode
-    allow_cache: False
+  # - type: editmode
```

Comment:
There wasn't really any reason, except that historically we sometimes kept some parts of the metafiles commented in case we would revert later on. But I agree with you that for better clarity these should be properly deleted. Done in the final commit. Thanks!